libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -360,7 +360,7 @@ dictionary LogEntry {
 dictionary InvoicePaidDetails {
     string payment_hash;
     string bolt11;
-    Payment? payment;
+    PaymentListItem? payment;
```

Comment:
An InvoicePaid event maybe shouldn't include A PaymentListItem, but only the data relevant to the paid invoice.